apiserver/params/params.go

```diff
@@ -801,15 +801,15 @@ type LogRecord struct {
 	Message  string    `json:"x"`
 }
 
-// GetBundleChangesParams holds parameters for making GetBundleChanges calls.
-type GetBundleChangesParams struct {
+// BundleGetChangesParams holds parameters for making Bundle.GetChanges calls.
+type BundleGetChangesParams struct {
```

Comment:
Why keep the "Get" in the name?  If you are changing the name anyway, probably better to remove the "Get" altogether.

BundelChangesParam, and BundleChangesResults.

While you are at it, BundleChangesChange -> BundleChange ?